indico/modules/events/series/__init__.py

```diff
@@ -0,0 +1,6 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2022 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
```

Comment:
otherwise-empty `__init__.py` files don't get a header

I just added a case for this to my header update script in master, so after rebasing you can run `python ./bin/maintenance/update_header.py` for the file to be updated (of course you could also do it manually but using the script it ensures the editor doesn't add e.g. a single newline)